ui/raidboss/data/06-ew/raid/p12s.txt

```diff
@@ -4,3 +4,169 @@ hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+# P1 Athena
+0.0 "--sync--" sync /Engage!/ window 0,1
+1.0 "--sync--" sync / 03:[^:]*:Hemitheos:00:5A:0{4}:00::12383:/ window 10,3 jump 1000.5 # Sync to P2 immediately through AddCombatant.
+11.1 "On the Soul" sync / 1[56]:[^:]*:Athena:8304:/ window 20,20
+21.2 "Paradeigma" sync / 1[56]:[^:]*:Athena:82ED:/
+24.3 "--sync--" sync / 1[56]:[^:]*:Athena:8315:/
+36.7 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/ window 1.5,0.5
+37.5 "White Flame" sync / 1[56]:[^:]*:Anthropos:82F0:/
+39.3 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/ window 1.5,0.5
+41.9 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/ window 1.5,0.5
+42.6 "White Flame" sync / 1[56]:[^:]*:Anthropos:82F0:/
+57.3 "Paradeigma" sync / 1[56]:[^:]*:Athena:82ED:/
+65.0 "Engravement of Souls" sync / 1[56]:[^:]*:Athena:8305:/
+74.8 "Searing Radiance/Shadowsear" sync / 1[56]:[^:]*:Anthropos:82F[12]:/ window 3,0.5
+75.8 "Astral/Umbral Glow" sync / 1[56]:[^:]*:Athena:830[CD]:/ window 3,0.5
```

Comment:
Because translations use full ability names (and different languages might abbreviate things differently), can you use the full ability names here (and elsewhere), e.g. `Astral Glow/Umbral Glow`.  Same with the line above, and saltire and cross, and apo/peri etc etc.

If you want to abbreviate when the timeline is displayed, you can use English "translations" by putting something in the `.ts` file, e.g. a `timelineReplace` section like this: https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/06-ew/raid/p8s.ts#L2714-L2732

Reply:
I've gone ahead and completed these suggestions.